modules/virt-configuring-vm-eviction-strategy-cli.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-configuring-live-migration.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-vm-eviction-strategy-cli_{context}"]
+= Configuring an eviction strategy by using the command line
+
+You can configure the `LiveMigration` eviction strategy for virtual machines (VMs) by using the command line.
```

Comment:
What about the cluster-wide `EvictionStrategy` setting?

Reply:
This doc story is for improving current documentation (updating 4.13). Cluster wide eviction strategy is a 4.14 TP feature. It does not appear to have a docs story, just RN, so I can add it here.